activity_calendar/admin.py

```diff
@@ -111,3 +113,5 @@ def recurrence_id_with_day(self, obj):
     inlines = [ParticipantInline]
 
 admin.site.register(ActivitySlot, ActivitySlotAdmin)
+admin.site.register(Calendar)
```

Comment:
Adding an `Inline` here for `CalendarActivityLink` makes it way easier to manage this in the admin panel.